snmp_f5/assets/dashboards/f5-networks.json

```diff
@@ -0,0 +1 @@
l":"","include_zero":true,"min":"auto","max":"auto"},"markers":[]},"layout":{"x":6,"y":3,"width":6,"height":3}}]}},{"id":3955856333376728,"definition":{"title":"UDP","title_align":"center","type":"group","background_color":"vivid_green","layout_type":"ordered","widgets":[{"id":5358818613441946,"definition":{"title":"# Open UDP Connections","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"q":"avg:snmp.sysUdpStatOpen{$snmp_device,$snmp_host} by {snmp_device,snmp_host}","on_right_yaxis":false,"style":{"palette":"cool","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"scale":"linear","label":"","include_zero":true,"min":"auto","max":"auto"},"markers":[]},"layout":{"x":0,"y":0,"width":6,"height":3}},{"id":1809244021974648,"definition":{"title":"# Established UDP Connections","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"q":"avg:snmp.sysUdpStatConnects{$snmp_device,$snmp_host} by {snmp_device,snmp_host}.as_count()","style":{"palette":"green","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"scale":"linear","label":"","include_zero":true,"min":"auto","max":"auto"},"markers":[]},"layout":{"x":6,"y":0,"width":6,"height":3}},{"id":716842456628528,"definition":{"title":"# UDP Connection Failures","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"q":"avg:snmp.sysUdpStatConnfails{$snmp_device,$snmp_host} by {snmp_device,snmp_host}.as_count()","on_right_yaxis":false,"style":{"palette":"warm","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"scale":"linear","label":"","include_zero":true,"min":"auto","max":"auto"},"markers":[]},"layout":{"x":0,"y":3,"width":6,"height":3}},{"id":8295219661336866,"definition":{"title":"# UDP Connections Not Accepted","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"q":"avg:snmp.sysUdpStatAcceptfails{$snmp_device,$snmp_host} by {snmp_device,snmp_host}.as_count()","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"scale":"linear","label":"","include_zero":true,"min":"auto","max":"auto"},"markers":[]},"layout":{"x":6,"y":3,"width":6,"height":3}}]}}],"template_variables":[{"name":"snmp_device","default":"*","prefix":"snmp_device"},{"name":"snmp_host","default":"*","prefix":"snmp_host"},{"name":"virtual_server","default":"*","prefix":"server"},{"name":"node","default":"*","prefix":"node"},{"name":"pool","default":"*","prefix":"pool"}],"layout_type":"ordered","is_read_only":false,"notify_list":[],"reflow_type":"fixed","id":"579-ci2-y23"}
```

Comment:
Typo x2: `"Byes` should be `"Bytes`. 
Reviewing this without newlines/structure was challenging. Is that something that can be fixed?